Bottom: e41bc2f7cfca1df669d883fbb90d8d436f20c4ac
Top:    e41bc2f7cfca1df669d883fbb90d8d436f20c4ac
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-04 22:56:59 +0000

Try adding polygon objects after each clipping stage. rtree->faster??

*** ACTUALLY SLOWS STARTUP DOWN ***







---


